py/picca/fitter2/pk.py

```diff
@@ -423,13 +423,15 @@ def pk_hcd_Rogers2018_uv_cross(k, pk_lin, tracer1, tracer2, **kwargs):
 def pk_gauss_smoothing(k, pk_lin, tracer1, tracer2, **kwargs):
     """
     Apply a Gaussian smoothing to the full correlation function
-
+    Args : k, sigma_smooth par and per
+    return : P(k) multiplied by G(k)^2
+    with G(k) = exp(-kpar^2 sigma_par^2 - kperp^2 sigma_perp^2)
     """
     kp  = k*muk
     kt  = k*np.sqrt(1.-muk**2)
     st2 = kwargs['per_sigma_smooth']**2
     sp2 = kwargs['par_sigma_smooth']**2
-    return sp.exp(-(kp**2*sp2+kt**2*st2)/2.)
+    return sp.exp(-(kp**2*sp2+kt**2*st2)/2.)**2
```

Comment:
We should use numpy instead of scipy here, for consistency with the rest of the Picca.

Also, why do you divide by 2 the exponent, and then square it? Couldn't you use the definition of G(k) as described in the docstring?